data/rewards/store.go

```diff
@@ -0,0 +1,95 @@
+package rewards
+
+import (
+	"github.com/Oneledger/protocol/data/balance"
+	"github.com/Oneledger/protocol/data/keys"
+	"github.com/Oneledger/protocol/serialize"
+	"github.com/Oneledger/protocol/storage"
+	"strings"
+)
+
+type RewardStore struct {
+	State  *storage.State
+	szlr   serialize.Serializer
+	prefix []byte
+
+	rewardOptions Options
```

Comment:
We should use a pointer to the store instead , just to follow the convention we already have